README.md

```diff
@@ -92,14 +92,26 @@ sudo apt-get install tezos-client-babylonnet
 
 ## Fedora (Red Hat) usage
 
+
+### Install `.rpm` package
+
 Build or download `.rpm` file from the CI and double-click on it or run:
 ```
 sudo yum localinstall <path to the rpm file>
 ```
+### Use copr package with `tezos-client`
+
+Also if you are using Fedora you can use Copr in order to install `tezos-client`.
+In order to do that run the following commands:
+```
+sudo yum copr enable @Serokell/Tezos
```

Comment:
Apparently there is `dnf` tool which is a modern replacement of `yum`: https://docs.pagure.org/copr.copr/how_to_enable_repo.html#how-to-enable-repo

Shouldn't we mention both?

Reply:
I haven't tried dnf but I think it should work as well